java/libraries/io/src/processing/io/LED.java

```diff
@@ -141,7 +141,7 @@ public void close() {
    *  Sets the brightness
    *  @param bright 0.0 (off) to 1.0 (maximum)
    */
-  public void set(float bright) {
+  public void setBrightness(float bright) {
```

Comment:
I initially picked set() because there's also a clear() that undos the user's actions and restores the initial value. But brightness() sounds good to me. Perhaps together with reset()?